aspnetcore/includes/benefits.md

```diff
@@ -1,21 +1,85 @@
 ASP.NET Core provides the following benefits:
 
-* A unified story for building web UI and web APIs.
+:::moniker range=">= aspnetcore-6.0"
+
+<!-- AUTHOR NOTE: >=6.0 content showcases Blazor and 
+                  Minimal APIs, demotes RP/MVC -->
+
+* A unified story for building web apps, web APIs, Azure IoT (Internet of Things) apps, and mobile backends.
+* Architected for testability.
+* [Blazor](xref:blazor/index) lets you create rich interactive client-side UIs using [.NET](/dotnet/standard/tour) and [C#](/dotnet/csharp/) with wide browser support based on HTML/JavaScript, including mobile browsers. You can also build hybrid desktop and mobile apps with .NET and Blazor.
+* [Minimal APIs](xref:fundamentals/minimal-apis) are a simplified approach for building fast web APIs with minimal code and configuration by fluently declaring API routes and actions.
+* Supports [Razor Pages](xref:razor-pages/index) and [Model-View-Controller (MVC)](xref:mvc/overview) app development.
+* Ability to develop and run on Windows, macOS, and Linux.
+* Open-source and [community-focused](https://live.asp.net/).
```

Comment:
These two bullets seem stuck in the middle the various ASP.NET Core frameworks (Blazor, Minimal APIs, SignalR, gRPC). We move them so that all the frameworks are listed together?